code/io/joy_ff-sdl.cpp

```diff
@@ -727,7 +727,7 @@ void joy_ff_afterburn_on()
 		SDL_HapticStopEffect(haptic, pAfterburn2.id);
 
 		pAfterburn2.eff.periodic.length = SDL_HAPTIC_INFINITY;
-		pAfterburn2.eff.periodic.magnitude = 0x3fff;
+		pAfterburn2.eff.periodic.magnitude = 0x1999;
```

Comment:
It was probably done to reduce the amount of used effects. In any case, I'm not going to poke around in code that I can test and it would be out of scope for this PR anyway.